src/style-sheets/generic/_card.scss

```diff
@@ -1,9 +1,83 @@
 .card-module {
   margin: 10px;
-  min-width: 235px;
-  min-height: 402px;
 
   background-color: $olLTPrimary;
   box-shadow: 0px 4px 4px rgba(60, 129, 137, 0.25);
   border-radius: 4px;
+  display: flex;
```

Comment:
this card implementation is very specific to the projects card. This file is more to keep generic styling for all cards. Could you create a new file for the _project-card.scss and move the specific changes there?